Components/LoginScreen.js

```diff
@@ -3,29 +3,54 @@ import { StyleSheet, TextInput, Text, View, Button, Image, AsyncStorage } from '
 import { createStackNavigator } from 'react-navigation';
 import logo from './images/logo_planted.png';
 import { connect } from 'react-redux';
-import {submitUserLoginInformation} from './helperFunctions/Login';
+import {submitUserLoginInformation, setLoginToAsyncStorage} from './helperFunctions/Login';
 import {updateToken, updateIsUserLoggedIn} from '../actions';
 
+
 class LoginScreenDumb extends Component {
     constructor(props) {
         super(props);
-        this.state = { email: '', password: '' };
+        this.state = { 
+            email: '', 
+            password: '',
+            failedToLogin: false,
+        };
       }
+    
+    //testing mode/ delete after
+    componentDidMount() {
+        let {navigation, dispatch} = this.props;
+        let userEmail = 'test1@gmail.com';
+        let userPassword = '1111';
+        submitUserLoginInformation(userEmail, userPassword)
```

Comment:
Why are we hardcoding the username and password?

Reply:
So that I don't have to type it every time I make a small change and want to see the profile page. We can comment it out for now and delete it later